### ISSUE TITLE
[7.0] Make each module transformer standalone

### ISSUE THREAD

**Issue by loganfsmyth** [babel/babel#5109]
Currently many of the module transforms explicitly do

    inherits: require("babel-plugin-transform-es2015-modules-commonjs"),

and then transform the CommonJS output from that into their respective module formats. Not only is this hard to follow, but it is confusing because people can technically write CommonJS `require` calls in their code and have them transformed into AMD or UMD for instance, even though they were not originally ES6 modules.

**Comment by hzoo** [comment]
this related to https://github.com/babel/babel/pull/3585 having to be 7.0 now

**Comment by hzoo** [comment]
Isn't this fixed now too https://github.com/babel/babel/pull/6230?

**Comment by loganfsmyth** [comment]
Yup!
